task-list/app/views/tasks/index.html.erb

```diff
@@ -24,26 +23,26 @@
     <tbody>
       <% @tasks.each do |task| %>
       <tr>
-        <td class="bigger_checkbox">
+        <td class="bigger_checkbox" class="bigger">
```

Comment:
I think in this case the `class="bigger"` attribute might override the previous attribute definition for class (`class="bigger_checkbox"`).
Either way, you can and should combine multiple classes for the same tag into a single attribute definition: `class="bigger_checkbox bigger"`.